test/bdd/features/erc725profile.feature

```diff
@@ -20,3 +20,11 @@ Feature: ERC725 Profile features
     And the 1st node's spend all the Ethers
     And I start the node
     Then the 1st node should start normally
+
+  Scenario: Provide own ERC725 identity and expect node to create profile
+    Given I setup 1 node
+    When I manually create ERC725 identity for 1st node
+    And I use the created ERC725 identity in 1st node
+    And I start the node
+    Then the 1st node should have a valid ERC725 identity
+    And the 1st node should have a valid profile
```

Comment:
If we could add a check, smth like:
And the 1st node should have skipped createProfile step
where we assert that following two events are notices in node's console log:

notify - Identity has already been created for node xxx. Identity is xxx.
notify - Profile has already been created for node xxx

then, this would be hell of a test case.

Reply:
I wasn't so interested what's happening behind the hood, I was more interested in outcome. Which is 'if i provide ERC 725 profile the node should start'. So far it crashed.
Maybe something like following is missing at the end: 'And the node should use manually created'.